apps/ccf-rui/src/app/shared/components/rotation-slider/slider-box/slider-box.component.html

```diff
@@ -1,14 +1,14 @@
-@if (displayedSlider !== sliderAxis) {
-  <mat-form-field
-    class="rotation-value"
-    subscriptSizing="dynamic"
-    (click)="displaySlider.emit()"
-    hraPlainTooltip="Rotation is measured in degrees"
-  >
-    <mat-label class="text form-input-label">{{ sliderAxis.toUpperCase() }}</mat-label>
-    <input matInput [value]="rotation[sliderAxis]" />
-  </mat-form-field>
-} @else {
+<mat-form-field
+  class="rotation-value"
+  subscriptSizing="dynamic"
+  (click)="showSlider()"
+  hraPlainTooltip="Rotation is measured in degrees"
+>
+  <mat-label class="text form-input-label">{{ sliderAxis.toUpperCase() }}</mat-label>
+  <input matInput [value]="rotation[sliderAxis]" />
+</mat-form-field>
+
+<ng-template #sliderContent>
```

Comment:
Use [CdkConnectedOverlay](https://material.angular.io/cdk/overlay/api#CdkConnectedOverlay) instead of manually managing the overlay. The reason we did it manually in apps.humanatlas.io is because the overlays were positioned globally instead of connected to an existing element